dimagi/pages/static/style/pages/blog.scss

```diff
@@ -53,7 +50,6 @@ a.post {
   img {
     margin-bottom: $space-small;
   }
-
```

Comment:
why was this space deleted? there should be a space between class definitions in `scss`. this is in the lint file in the root of the project